docker/gae-run-app.sh

```diff
@@ -1,4 +1,18 @@
 #!/usr/bin/env bash
+# Copyright 2019 Google Inc.
```

Comment:
I think the Docker files were primarily written by someone outside Google, so I don't think they should have a Google copyright line.

Reply:
Okay. Will remove this. But even if it was written by someone else they would have signed the CLA and it's a Google copyright now right?

Reply:
The person would have signed a CLA, and that certainly means we can open-source it and put it in the repo, but I'm not 100% sure that means Google can claim copyright on it.

Reply:
Ack. Removed license from docker files.